### PR TITLE
Fibonacci Search Added

### PR DESCRIPTION
**Pull request by ianjathanna**
This Pull Request resolves Issue #19 #1 #24 

Description:
In this Pull Request we have done the following additions/edits for the code:

### CODE REVIEW

File: Fibonacci_Search/ianjathanna.c [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    return -1;
+}
+
+int main(void)
+{
+    int arr[] = {10, 22, 35, 40, 45, 50, 80, 82,
+                 85, 90, 100};
+    int n = sizeof(arr)/sizeof(arr[0]);
+    int x = 85;
```
Comment (MJ10) [inline Fibonacci_Search/ianjathanna.c]: Accept the input from user instead of hardcoding the values.

File: PARTICIPANTS/ianjathanna.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Name: Ian Jathanna
```
Comment (MJ10) [inline PARTICIPANTS/ianjathanna.txt]: The directory name should be `Participants`

**Comment by salman-bhai** [comment]
Are there any updates or should we close this PR as `spam`?

**Comment by salman-bhai** [comment]
spam

**Pull request closed without merging**